.evergreen/config.yml

```diff
@@ -974,18 +974,26 @@ functions:
           ${PREPARE_SHELL}
 
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
-          source activate_venv.sh || :
```

Comment:
This pipe was being used to prevent errors on the 3.0 1404 ubunutu version, which doesn't appear to have venv installed. To get around this, we check to see if it's installed before running activate_venv, since `source` is not cygwin-friendly.